recommendation/tools.md

```diff
@@ -84,5 +84,10 @@ and the scope may identify a particular stream of development that that tool is
       <td>Jest</td>
       <td><a href="https://jestjs.io/" rel="nofollow">https://jestjs.io/</a></td>
     </tr>
+    <tr>
+      <td>E2E testing</td>
```

Comment:
E2E testing would be it's own type of recommendation (It's not Unit Testing).
The scope for now would be nothing as long as it's able and expected to be used on all projects.